### PR TITLE
Add missing direct dependencies

### PR DESCRIPTION
**Pull request by samoht**
Found by https://github.com/ocurrent/opam-dune-lint

### CODE REVIEW

File: bench/irmin-pack/dune [hunk 1]
```diff
@@ -1,8 +1,10 @@
 (executables
  (names main layers)
+ (public_names bench-pack bench-pack-layers)
+ (package irmin-bench)
  (preprocess
   (pps ppx_deriving_yojson ppx_repr))
- (libraries irmin-pack irmin-test.bench irmin-layers lwt unix cmdliner logs
+ (libraries irmin-pack irmin-test.bench irmin-layers lwt cmdliner logs
```
Comment (CraigFe) [inline bench/irmin-pack/dune:7]: That one is unfortunate: pretty sure it's needed by `(implicit_transitive_deps false)`, so this is a bit of a step backwards.

**Review by CraigFe — APPROVED** [review]

File: bench/irmin-pack/dune (hunk 1 above)
Comment (samoht) [reply bench/irmin-pack/dune]: I've reverted that change

**Comment by samoht** [comment]
Thanks to @talex5 the CI is now happy :-)

**Comment by samoht** [comment]
Merging this to unbreak the CI.

**Pull request merged**